### PR TITLE
[MTM-50975] Release notes: optional configuration in SSO page, allowi…

### PR DESCRIPTION
**Pull request by mbak-c8y**
Release notes: optional configuration in SSO page, allowing displaying error from the SSO flow in UI error page, instead of plain HTML as it was before.

Before: when an error occurs in the SSO auth flow the plain HTML is returned in the browser.
Now: Optional configuration was added which allows control flow by UI and in case of any error it can be printed in a pretty error page.

Why Optional? It required an update in the authorization server.
It will not affect the current SSO configuration.

### CODE REVIEW

**Review by pink-mongoose — COMMENTED** [review]
What do we mean by a _proper error _page__

**Comment by BeateRixen** [comment]
> Release notes: optional configuration in SSO page, allowing displaying error from the SSO flow in UI error page, instead of plain HTML as it was before.
> 
> Before: when an error occurs in the SSO auth flow the plain HTML is returned in the browser. Now: Optional configuration was added which allows control flow by UI and in case of any error it can be printed in a pretty error page.
> 
> Why Optional? It required an update in the authorization server. It will not affect the current SSO configuration.

@mbak-c8y This is exactly the information we should add to the release notes entry :-). I will make a suggestion.

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: q3kclF6pO
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-50975
+version: 10.20.499.0
+---
+SSO configuration has been extended with the optional: `Redirect to the user interface application`.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:17]: Previously, when an error occurred on logging in via SSO, the plain HTML error text was displayed in the browser. With this change, optional configuration has been added which allows displaying the error text as a standard UI error message. The new configuration option is optional and does not affect current SSO configurations.

Comment (BeateRixen) [reply content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:17]: I would change it to the following text for the users and have the details as a change log in the operations release notes. What do you think, @pink-mongoose @mbak-c8y?

Comment (mbak-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:17]: I do not think Operations release is required, it is rather tenant administrator change.

Comment (mbak-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md]: I rephrased it based on comment, @BeateRixen  is it sound better now?

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+date: ""
+title: Single-sign-on improve error information page
+product_area: Platform services
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Improvement
```
Comment (kgol-c8y) [inline content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:7]: Isn't this a feature and not improvement?

**Review by pink-mongoose — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    label: Authentication
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-50975
+version: 10.20.499.0
+---
+Previously, when an error occurred on logging in via SSO, the plain HTML error text was displayed in the browser.
+With this change, optional `Redirect to the user interface application` configuration has been added which allows displaying the error text as a standard UI error message.
+The new configuration option is optional and does not affect current SSO configurations.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:19]:
```suggestion
The new configuration parameter is optional and does not affect current SSO configurations.
```

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-50975
+version: 10.20.499.0
+---
+Previously, when an error occurred on logging in via SSO, the plain HTML error text was displayed in the browser.
+With this change, optional `Redirect to the user interface application` configuration has been added which allows displaying the error text as a standard UI error message.
+The new configuration parameter is optional and does not affect current SSO configurations.
+Using this option requires updating the "Valid Redirect URIs" in the authorization server with the value "<tenant_domain>/apps/*".
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:20]:
```suggestion
Using this option requires updating the "Valid Redirect URIs" in the authorization server with the value "<tenant_domain>/apps/*". For details, refer to [Custom template configuration](/authentication/sso/custom-template).
```

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+date: ""
+title: Single-sign-on improve error information page
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:3]:
```suggestion
title: Improved error information for single sign-on login
```

**Review by pink-mongoose — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md (hunk 2 above)
Comment (mbak-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-499-0-single-sign-on-improve-error-information-page.md:7]: improvement

**Review by BeateRixen — APPROVED** [review]

**Review by kgol-c8y — APPROVED** [review]

**Pull request merged**